docs/DOCUMENTATION.md

```diff
@@ -120,6 +120,8 @@ Events are also triggered at different points in the render method:
 
 It is rare, and discouraged to override the render method. Instead, use the available hooks to control the render logic.
 
+The ```updateDOM``` method is called during step 3 during DOM creation and by default, invokes: ```this.templateRender(this.$el, this.template, this.prepare());```. If you wish to have custom logic, override this method.
+
```

Comment:
make a note that listView does more complicated things so be careful when overriding a from ListView?